### PR TITLE
Add 3.2 reference documentation

### PR DESCRIPTION
**Pull request by jyemin**

### CODE REVIEW

File: docs/reference/content/driver-async/getting-started/quick-tour-admin.md [hunk 1]
```diff
@@ -223,7 +223,7 @@ For more information about text search see the [text index]({{< docsref "/core/i
 ## Running a command
 
 While not all commands have a specific helper, however you can run any [command]({{< docsref "/reference/command">}})
-by using the [`runCommand()`](http://api.mongodb.org/java/3.1/?com/mongodb/async/client/MongoDatabase.html#runCommand-org.bson.conversions.Bson-com.mongodb.ReadPreference-com.mongodb.async.SingleResultCallback-) 
+by using the [`runCommand()`](http://api.mongodb.org/java/3.2/?com/mongodb/async/client/MongoDatabase.html#runCommand-org.bson.conversions.Bson-com.mongodb.ReadPreference-com.mongodb.async.SingleResultCallback-)
```
Comment (rozza) [inline docs/reference/content/driver-async/getting-started/quick-tour-admin.md]: Update to use apiref - so we dont have to tweak the version each time. eg: 

    [`runCommand()`]({{< apiref com/mongodb/async/client/MongoDatabase.html#runCommand-org.bson.conversions.Bson-com.mongodb.ReadPreference-com.mongodb.async.SingleResultCallback-" >}})

File: docs/reference/content/driver/getting-started/quick-tour-admin.md [hunk 2]
```diff
@@ -166,7 +166,7 @@ For more information about text search see the [text index]({{< docsref "/core/i
 ## Running a command
 
 While not all commands have a specific helper, however you can run any [command]({{< docsref "/reference/command">}})
-by using the [`runCommand()`](http://api.mongodb.org/java/3.1/?com/mongodb/client/MongoDatabase.html#runCommand-org.bson.conversions.Bson-com.mongodb.ReadPreference-) 
+by using the [`runCommand()`](http://api.mongodb.org/java/3.2/?com/mongodb/client/MongoDatabase.html#runCommand-org.bson.conversions.Bson-com.mongodb.ReadPreference-)
```
Comment (rozza) [inline docs/reference/content/driver/getting-started/quick-tour-admin.md]: Prefer apiref helper

File: docs/reference/content/driver/reference/index.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ title = "Reference"
 ## Reference
 
 The reference documentation for the Java driver focuses on high-level documentation and use-cases. The Javadoc-generated API reference is
-[here](http://api.mongodb.org/java/3.1/).    
+[here](http://api.mongodb.org/java/3.2/).
```
Comment (rozza) [inline docs/reference/content/driver/reference/index.md]: apiref helper might even work here! - not sure though.

File: docs/landing/static/versions.json [hunk 4]
```diff
@@ -1 +1 @@
-[{"version": "3.1"}, {"version": "3.0"}, {"version": "2.13"}]
+[{"version": "3.2"}, {"version": "3.1"}, {"version": "3.0"}, {"version": "2.13"}]
```
Comment (rozza) [inline docs/landing/static/versions.json]: We'll need 2.14 as well.

File: docs/landing/data/releases.toml [hunk 5]
```diff
@@ -19,19 +25,19 @@ current = "3.1.1"
...
   name = "mongodb-driver"
   description = "The synchronous driver, new in 3.0.<br/>For older versions of the driver or for OSGi-based applications please use the `mongo-java-driver`."
-  versions = "3.1.1,3.0.4"
+  versions = "3.2.0-rc0,3.1.1,3.0.4"
 
 [[drivers]]
   name = "mongo-java-driver"
   description = "An uber jar containing the bson library, the core library and the mongodb-driver.<br/>This artifact is a valid OSGi bundle."
-  versions = "3.1.1,3.0.4,2.13.3"
+  versions = "3.2.0-rc0,3.1.1,3.0.4,2.13.3"
```
Comment (rozza) [inline docs/landing/data/releases.toml]: We'll need 2.14 as well here.

File: docs/reference/content/driver/getting-started/quick-tour-admin.md (hunk 2 above)
Comment (jyemin) [inline docs/reference/content/driver/getting-started/quick-tour-admin.md]: Done

File: docs/landing/static/versions.json (hunk 4 above)
Comment (jyemin) [inline docs/landing/static/versions.json]: Done

File: docs/landing/data/releases.toml (hunk 5 above)
Comment (jyemin) [inline docs/landing/data/releases.toml]: Done

File: docs/reference/content/driver-async/getting-started/quick-tour-admin.md (hunk 1 above)
Comment (jyemin) [inline docs/reference/content/driver-async/getting-started/quick-tour-admin.md]: Done

File: docs/reference/content/driver/reference/index.md (hunk 3 above)
Comment (jyemin) [inline docs/reference/content/driver/reference/index.md]: Done.  The link is a bit funny (http://api.mongodb.org/java/3.2/?), but it works.

File: docs/landing/data/releases.toml (hunk 5 above)
Comment (rozza) [inline docs/landing/data/releases.toml]: I'm not sure this has been pushed yet.

**Comment by rozza** [comment]
Once you are happy that its also listing 2.14.0 on the landing page its a LGTM.

File: docs/landing/data/releases.toml (hunk 5 above)
Comment (jyemin) [inline docs/landing/data/releases.toml]: I had added a version for it above.  Now it's in this versions list as well.

**Pull request closed without merging**